### ISSUE TITLE
[Bug] Cannot use with out a proxy

### ISSUE THREAD

**Issue by artiga033** [krillinai/KrillinAI#16]
config:
```toml
[app]
    #proxy = ""
```
logs:
```
2024-12-27T22:07:37.322+0800    info    util/base.go:160        已找到ffmpeg
2024-12-27T22:07:37.322+0800    info    util/base.go:220        已找到ffprobe
2024-12-27T22:07:37.373+0800    info    util/base.go:283        没有找到yt-dlp，即将开始自动安装
2024-12-27T22:07:37.373+0800    error   util/base.go:308        下载yt-dlp失败  {"error": "Get \"https://github.com/yt-dlp/yt-dlp/releases/download/2024.12.13/yt-dlp_linux\": proxyconnect tcp: dial tcp :0: connect: connection refused"}
2024-12-27T22:07:37.373+0800    error   KrillinAI/main.go:42    yt-dlp环境准备失败      {"error": "Get \"https://github.com/yt-dlp/yt-dlp/releases/download/2024.12.13/yt-dlp_linux\": proxyconnect tcp: dial tcp :0: connect: connection refused"}
```
it is the same behavior whether explicitly set proxy to `""` or comment it out.

The reason may be that in Golang empty string can be successfully parsed into a zero-valued URL instead of nil.

**Comment by maranello-o** [comment]
I've pushed a fix to `feat-aitool` branch, ([d4a01d6](https://github.com/krillinai/KrillinAI/commit/d4a01d6818b4497043209603bd55afd7273b86e1))
you can test by:
```
git fetch
git checkout feat-aitool
go run main.go
```
thanks!

**Comment by artiga033** [comment]
works as expected
